presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt

```diff
@@ -15,20 +15,303 @@
  */
 package com.android254.presentation.about.view
 
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.*
+import androidx.compose.material3.Icon
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.android254.presentation.common.theme.DroidconKE2022Theme
+import com.android254.presentation.R
+
+val isdark = mutableStateOf(false)
 
 @Composable
 fun AboutScreen() {
-    Text(text = "About Screen")
+
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(if (isdark.value) Color.Black.copy(.6f) else Color.White.copy(.6f))
```

Comment:
@wangerekaharun  Quick one

Does the theme setup on the repo automatically choose color depending on if the device is set to dark mode or not?

Or do we need to do this?

`if (isdark.value) Color.Black.copy(.6f) else Color.White.copy(.6f)`

Reply:
It automatically chooses so no need to have this

Reply:
It automatically chooses so no need to have this

Reply:
Hi team,
I've made changes to color it's now autopicking